support/hashgraph.js

```diff
@@ -137,12 +143,48 @@ const CallContract = async (client, {
   return contractReceipt.status.toString() === 'SUCCESS'
 }
 
+// My bit, it's not as spicey as smithies.
```

Comment:
Remove comment and add something more descriptive, why using records, why did we decide to seaprate the methods etc

Reply:
Added documentation latest commit - insight into functionality, required parameters and what is returned.